src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/event/EventToSendProcessorTest.java

```diff
@@ -60,7 +61,8 @@ public void setup() {
 
   private EventToSend createEvent(String id) {
     String payload = FixtureHelper.loadPackageObjectNode("SurveyLaunchResponse").toString();
-    return new EventToSend(UUID.fromString(id), TopicType.SURVEY_LAUNCH.name(), payload);
+    return new EventToSend(
+        UUID.fromString(id), TopicType.SURVEY_LAUNCH.name(), payload, LocalDateTime.now());
```

Comment:
since the unit tests don't test this (time ordering) functionality , we will probably need a cucumber test.